### DISCUSSION TITLE
Cannot load multiplayer save with specific character after switching to permadeath

### DISCUSSION THREAD

**Discussion by Mixmaster49** [FakeFishGames/Barotrauma#15137] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In a multiplayer permadeath campaign, one of the players died when permadeath was mistakenly left off. After we had turned on permadeath, and the player joined the game mid-round, the next level would not load and it would put us into the lobby. This only occurred when the player was in the lobby at the time of loading, and was only resolved by getting the player to create a new character in the lobby. 

### Reproduction steps

1. Start a multiplayer campaign with permadeath off
2. Kill one of the players and have them leave the game
3. Complete the mission with the player still out of the game
4. Go back to the lobby, turn on permadeath and have the player join
5. Try to reload the campaign, it should send you back to lobby as long as the player is in the game

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I gave this a test, but unfortunately could not get the issue to happen at my end (the round seems to start with no issues). So I suspect there were some other factors at play here that caused the bug to happen. Do you know for example what option the player chose when they died, waiting for the next round or respawning immediately? If you still happened to have a save file + the character data file available where this occurs, it would most likely help us a lot in diagnosing this. Thanks and sorry for the inconvenience!

**Reply by Mixmaster49** [reply]
[Multiplayer.zip](https://github.com/user-attachments/files/17742871/Multiplayer.zip)

These are the current save and backup files for the campaign. The problem was with the character "Bingo Crowabr" (who is usually Bingo Crowbar). Of note is that for some reason the character file of the earliest backup has Bingo as their newly controlled character, even though they had not switched at that time (I checked the last modified timestamps on the files and referenced with server log timestamps to confirm). I remember that the game said a player was waiting for respawn, but I do not know of any options the player picked.
